lib/client/doc.js

```diff
@@ -648,6 +652,9 @@ Doc.prototype._otApply = function(op, options) {
       return;
     }
 
+    if (!source && this.type === types.defaultType && op.op.length > 1) {
```

Comment:
@Nodeigi Should it only be for remote ops (`!source`)? Couldn't the same issue with not transforming each component also happen for only local ops?

Reply:
I suppose that it's safe for local operations. Before my changes, `source` was the main criteria whether to split ops array and emit it individually or not.